### ISSUE TITLE
what is valid metric ?  

### ISSUE THREAD

**Issue by zhijiejia** [open-mmlab/mmselfsup#306]
What are the metrics used by the comparative learning algorithm in the verification stage? Can the nce loss on the verification set be used as the verification loss of the verification set? I feel that adding a linear layer in the verification phase will affect the training time too much. So I'd like to ask you how you do it？？

**Comment by fangyixiao18** [comment]
Generally, we only monitor the loss curve in pre-train step as we don't have 'accuracy' during pre-train.
Or you can use offline KNN test to monitor the KNN acc, which might help.
https://github.com/open-mmlab/mmselfsup/blob/master/docs/en/tutorials/6_benchmarks.md#imagenet-nearest-neighbor-classification
